Automattic/pocket-casts-android
# Bookmarks - Add delete bookmark action in notification

## Description

This adds a delete bookmark action in "Bookmark added" notification


Fixes #1595 

## Testing Instructions

#### Test.1 Bookmark deleted
1. Go to `Profile` -> `Settings` -> `Headphone controls`
2. Tap `Next action`
3. Select `Add bookmark`
4. Play any podcast episode
5. Minimize the app
6. Double-tap the headphone to add a bookmark
7. Notice that `Bookmark added` notification is displayed
8. Tap `Delete` in the notification
9. ✅ Notice that bookmark is deleted and "Bookmark deleted" snackbar is displayed

#### Test.2 Bookmark not found
1. Continue from above
2. Repeat Test.1 steps 4-7
3. Open the app and delete the newly added bookmark
4. Tap `Delete` in the notification
5. ✅ Notice that "Bookmark not found" snackbar is displayed

#### Test.3 Change title when bookmark episode not playing
** I also used `bookmarkUuid` in the notification intent to load an existing bookmark while changing the bookmark title in https://github.com/Automattic/pocket-casts-android/pull/1617/commits/602ceae25e67a5171a977a599f334e08e609ced9

To test this change,
1. Continue from above
2. Repeat Test.1 steps 4-7
3. Open the app and play another episode 
4. Tap `Change title` in the notification
5. ✅ Notice that "Change bookmark title" screen is shown

## Screenshots or Screencast 

<img width=300 src="https://github.com/Automattic/pocket-casts-android/assets/1405144/93a91958-f9d5-4d14-aff2-79f3cd2399b5"/>

## Checklist
- Any jetpack compose components I added or changed are covered by compose previews N/A
 
#### I have tested any UI changes...

- with different themes N/A
- with a landscape orientation N/A
- with the device set to have a large display and font size N/A
- for accessibility with TalkBack N/A
